ci-operator/step-registry/configure-registry-storage/deploy-s3-cloudfront/configure-registry-storage-deploy-s3-cloudfront-commands.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# This script will deploy an aws s3 bucket and cloudfront distribution.
+# It need deploy against aws platform.
+
+if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+    echo "Setting proxy"
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+case "${CLUSTER_TYPE}" in
+aws|aws-arm64|aws-usgov) export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred;;
+aws-c2s|aws-sc2s) export AWS_SHARED_CREDENTIALS_FILE=${SHARED_DIR}/aws_temp_creds;;
```

Comment:
I'm afraid this will not work on C2S/SC2S, the credential for C2S/SC2S is only valid for _one_ hour, which means it probably expired while this step is invoked since it was applied before cluster provision.

You need to apply a new one: https://github.com/openshift/release/blob/master/ci-operator/step-registry/aws/c2s/init-token-service/aws-c2s-init-token-service-commands.sh#L65-L78

Reply:
@yunjiang29 I will update c2s/sc2s part when the step stable. Thanks for the point